### ISSUE TITLE
identity preserving?

### ISSUE THREAD

**Issue by Yijunmaverick** [jerryli27/TwinGAN#17]
Thanks for your great work. I'm just wondering how do you measure or evaluate how well the identity is preserved in your results?

At least visually it is hard for me to recognize Emma in the anime results.

**Comment by jerryli27** [comment]
You're correct. I only measured identity preservation qualitatively by basically eyeballing and it's only a proof of concept with a lot of room for improvements.

An indirect way is to measure the distance between the embedding of the original image versus the generated anime version -- which we can track from training logs. Maybe I should include that number somewhere in the paper. Thanks for the suggestion!
